ci/prow/periodic_config.go

```diff
@@ -332,48 +333,55 @@ func generateGoCoveragePeriodic(title string, repoName string, _ yaml.MapSlice)
 // generateIssueTrackerPeriodicJobs generates the periodic jobs to automatically manage issue lifecycles.
 // It's a mirror of fejta bot - https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/test-infra/fejta-bot-periodics.yaml.
 func generateIssueTrackerPeriodicJobs() {
-	staleJobName := "ci-knative-issue-tracker-stale"
+	generateIssueTrackerPeriodicJobsForRepo("knative/test-infra", 90, 30, 30)
+	generateIssueTrackerPeriodicJobsForRepo("knative/docs", 90, 30, 30)
+}
+
+func generateIssueTrackerPeriodicJobsForRepo(repo string, daysToStale, daysToRot, daysToClose int) {
+	repoForJob := strings.Replace(repo, "/", "-", -1)
+	feedbackNote := "Send feedback to [Knative Productivity Slack channel](https://knative.slack.com/messages/CCSNR4FCH) or file an issue in [knative/test-infra](https://github.com/knative/test-infra)."
+	staleJobName := fmt.Sprintf("ci-%s-issue-tracker-stale", repoForJob)
 	staleLabelFilter := `
         -label:lifecycle/frozen
         -label:lifecycle/stale
         -label:lifecycle/rotten`
-	staleUpdatedTime := "2160h"
-	staleComment := `--comment=Issues go stale after 90d of inactivity.<br/>
-        Mark the issue as fresh with /remove-lifecycle stale.<br/>
-        Stale issues rot after an additional 30d of inactivity and eventually close.<br/>
-        If this issue is safe to close now please do so with /close.<br/><br/>
-        Send feedback to Knative Productivity Slack channel or knative/test-infra.<br/><br/>
-        /lifecycle stale`
-	generateIssueTrackerPeriodicJob(staleJobName, staleLabelFilter, staleUpdatedTime, staleComment)
+	staleUpdatedTime := fmt.Sprintf("%dh", daysToStale*24)
+	staleComment := fmt.Sprintf(`--comment=Issues go stale after *%dd* of inactivity.<br/>
+        Mark the issue as fresh by adding the comment <code>/remove-lifecycle stale</code>.<br/>
```

Comment:
Use backticks instead of `<code>`, here and everywhere. Also, since it's markdown, you should be able to use linefeeds instead of `<br/>` tags.

Reply:
The reason I used `<code>` is it's verbose to add backquote in a backquoted strings, see https://stackoverflow.com/questions/4423951/how-to-put-a-backquote-in-a-backquoted-string.
And the reason I used `<br/>` tags is because we remove the empty lines when we generate the final config, and the other configs will be broken if we remove this logic.

Let me know if you have other concerns.

Reply:
You can use double quotes and `\n`.

Reply:
Done.